### PR TITLE
Task J-137: + add link to episode

### PR DESCRIPTION
**Pull request by JuliaDzyuba**

### CODE REVIEW

**Review by EugeniusGR — DISMISSED** [review]

File: frontend/src/components/podcast/components/episode-table/helpers/get-rows.helper.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import { Episode } from 'common/types/types';
 import { formatDate } from './get-formatted-date';
 
-const getRows = (episodes: Episode[]): Record<string, string>[] => {
+const getRows = (episodes: Episode[]): Record<string, string >[] => {
   const rows = episodes.map((episode) => ({
     name: episode.name,
     description: episode.description,
     createdAt: formatDate(new Date(episode.createdAt)),
+    episodeId: String(episode.id),
```
Comment (ghost) [inline frontend/src/components/podcast/components/episode-table/helpers/get-rows.helper.ts:9]:
```suggestion
const getRows = (episodes: Episode[]): Record<string, unknown >[] => {
  const rows = episodes.map((episode) => ({
    name: episode.name,
    description: episode.description,
    createdAt: formatDate(new Date(episode.createdAt)),
    episodeId: episode.id,
```

File: frontend/src/components/common/table/table.tsx [hunk 2]
```diff
@@ -32,16 +34,23 @@ const Table: React.FC<Props> = ({ columns, data = [] }) => {
...
-          return (
+          const episodeId = row.original.episodeId;
+          return (            
             <tr {...row.getRowProps()} key={i} className={styles.episodeRow}>
               {row.cells.map((cell, i) => {
-                return (
-                  <td {...cell.getCellProps()} key={i}>
+                return ( i === 1 
+                  ? <td {...cell.getCellProps()} key={i}>
+                    <Link to={`${AppRoute.EPISODES}/${episodeId}`} key={i} className={styles.link}>
```
Comment (ghost) [inline frontend/src/components/common/table/table.tsx]: we shouldn't change it in the common table. we need to change the `getColumn` for episode function

**Review by ghost — CHANGES_REQUESTED** [review]

File: frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    {
+      Header: 'Name and Host',
+      accessor: (originalRow): Record<string, unknown> => {
+        const row = originalRow as Row;
+        return {
+          name: row.name,
+          episodeId: row.episodeId,
+        };
+      },
+      Cell: function getLink({ value }): JSX.Element  {
```
Comment (ghost) [inline frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.tsx]:
```suggestion
      Cell: ({ value }): JSX.Element =>  {
```

File: frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+const getColumns = (): Column[] => {
+  const columns: Column[] = [
+    {
+      Header: '#',
+      accessor: (_originalRow, rowIndex): number => {
+        return rowIndex + INCREASE_CONT_FOR_IDX;
+      },
+    },
+    {
+      Header: 'Name and Host',
```
Comment (ghost) [inline frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.tsx]:
```suggestion
      Header: 'Name',
```

File: frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
+import { Column } from 'react-table';
+import { INCREASE_CONT_FOR_IDX } from 'common/constants/constants';
+import { Link } from 'components/common/common';
+import { AppRoute } from 'common/enums/enums';
+
+type Row = {
+  name: string,
+  episodeId: string,
+};
```
Comment (ghost) [inline frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.tsx]:
```suggestion
type EpisodeNameRow = {
  name: string,
  episodeId: string,
};
```
move to `frontend/src/components/podcast/components/episode-table/types/`

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by yaroslav207 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**